Routes/Product listing page/product.html

```diff
@@ -0,0 +1,343 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Home</title>
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
you can directly import them in your `main.css` file (where you are importing CSS for all the components), then you won't need to link them in every page HTML

Reply:
thanks for pointing it out have addressed that